src/nvim/ex_docmd.c

```diff
@@ -2740,7 +2740,6 @@ static void undo_cmdmod(const exarg_T *eap, int save_msg_scroll)
 }
 
 
-
```

Comment:
If you do this unrelated change, remove 2 lines. ;P

Reply:
fixed